### PR TITLE
Update install

### PR DESCRIPTION
**Pull request by ssepi0l-pv**
Hello!
I wanted to commit these changes because I'd love to see official support for Arch Linux. I modified this script and installed Sliver on my system using it, so I know it works. Nevertheless, please do check it and try it yourselves! :)

### Details

As you can see, I added the INSTALLER variable, which stores the package manager in an array which will be used in case the user doesn't have awk. gpg or curl. After that, I added a new elif that will check the availability of the Pacman binary, and if it exists it will install the mingw-w64 packages for cross-compiling.

Apart from that, instead of exiting the script when awk,gpg,curl isn't found, now it'll install the missing packages.

Have a great day!!!!

### CODE REVIEW

**Review by rkervella — APPROVED** [review]

File: docs/install [hunk 1]
```diff
@@ -15,19 +15,25 @@ if command -v apt &> /dev/null; then
...
 else
     echo "Unsupported OS, exiting"
     exit
 fi
 
 # Verify if necessary tools are installed
 for cmd in curl awk gpg; do
-    if ! command -v "$cmd" &> /dev/null; then
-        echo "$cmd could not be found, exiting"
-        exit 1
```
Comment (rkervella) [inline docs/install:30]: Might be worth defaulting to `exit` here instead of trying to install it, in case the command name and the package name differ.

**Review by rkervella — CHANGES_REQUESTED** [review]

File: docs/install (hunk 1 above)
Comment (ssepi0l-pv) [reply docs/install:30]: Ok!

**Comment by ssepi0l-pv** [comment]
merge when? :3

**Comment by moloch--** [comment]
Sorry it's been a busy summer, and I've been on vacation for a few weeks but we'll try to get this merged shortly.

**Comment by ssepi0l-pv** [comment]
no worries! take your time :)

**Comment by ssepi0l-pv** [comment]
thank you! this is my first merge request ever!!! im very happy that it got approved :)
will be around and check what else i can do to help!!

**Pull request merged**